.gitignore

```diff
@@ -20,3 +20,6 @@
 
 # Ignore application configuration
 /config/application.yml
+
+# Ignore extra db
+/db/development.sqlite3.backup
```

Comment:
Normally I would say this isn't something to put in the .gitignore -- if a developer makes a backup, it's up to her/him to not check it in. But I think it's fine. Maybe update it to match all `/db/*.sqlite3.backup` files, for good measure, so you can have multiple copies, or backup other environments' databases, without adding more lines in .gitignore.